### PR TITLE
Adding Static/Dynamic TS Resources 

### LINKED ISSUE #2759: Create Resources for 2018 Public Transmittal Sheet
The task is complete with the following resources in the platform repository. 

Static/Dynamic Transmittal Sheet (TS)
- [x] 2018 TS CSV Data Dictionary 
- [x] 2018 TS CSV Schema
- [x] 2018 TS PDF Schema
- [x] 2018 TS CSV Header

### PR DESCRIPTION
**Pull request by MichaelFurey17**
Close #2759

### CODE REVIEW

File: docs/v2/spec/2018_Public_TS_Data_Dictionary.csv [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿Field Name,Field Data Type,Valid Values
```
Comment (Kibrael) [inline docs/v2/spec/2018_Public_TS_Data_Dictionary.csv]: For all the rows in valid values, please only list codes (numeric, NA, or Exempt). Descriptions of the data fields are handled elsewhere.

File: docs/v2/spec/2018_Public_TS_Schema.csv [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+Calendar Year ,Numeric,,Example: 2018
+Calendar Quarter,Numeric,4,Description: 4. Annual submissions for covered loans and applications with respect to which final action was taken during the period between January 1st (01/01) and December 31st (12/31)
+Contact Person's Office Street Address ,Character Varying,,Example: 4321 Random Blvd Ste 201
+Contact Person's Office City,Character Varying,,Example: Somecity
+Contact Person's Office State,Character Varying,,Example: ST
+Contact Person's Office Zip Code,Character Varying,,Example: 12345-1010
+Federal Agency,Numeric,1 2 3 5 7 9,Descriptions: 1. Office of the Comptroller of the Currency (OCC) 2. Federal Reserve System (FRS) 3. Federal Deposit Insurance Corporation (FDIC) 5. National Credit Union Administration (NCUA) 7. United States Department of Housing and Urban Development (HUD) 9. Consumer Financial Protection Bureau (CFPB)
+Total Number of Entries Contained in Submission,Numeric,,Example: 5000
+Federal Taxpayer Identification Number,Character Varying,,Example 99-9999999
+Legal Entity Identifier (LEI),Character Varying ,20 Character LEI,Example: 12121212121212121212
```
Comment (Kibrael) [inline docs/v2/spec/2018_Public_TS_Schema.csv]: Remove `20 Character LEI` from the valid values column, it is a description not a valid value.

File: docs/v2/spec/2018_Public_TS_Data_Dictionary.csv (hunk 1 above)
Comment (MichaelFurey17) [reply docs/v2/spec/2018_Public_TS_Data_Dictionary.csv]: Sure - changes will be reflected in the next commit.

File: docs/v2/spec/2018_Public_TS_Schema.csv [hunk 3]
```diff
@@ -0,0 +1,13 @@
+﻿Field Name,Field Data Type,Valid Values,Descriptions and Examples
```
Comment (Kibrael) [inline docs/v2/spec/2018_Public_TS_Schema.csv]: Can you rename this file from Schema to data_dictionary per our conversation today?

Comment (Kibrael) [inline docs/v2/spec/2018_Public_TS_Schema.csv]: Please change "descriptions and examples" to "notes". 

If descriptions or examples are needed users can refer to the larger file. I'm seeing a notes column as explaining odd edge cases (of which I can't think of one for the TS).

Please also remove the examples. The names of the fields along with the data types are sufficient in this context to explain what should be present in the file.

File: docs/v2/spec/2018_Public_TS_Header.csv [hunk 4]
```diff
@@ -0,0 +1 @@
+﻿Record Identifier,Financial Institution Name,Calendar Year ,Calendar Quarter,Contact Person's Office Street Address ,Contact Person's Office City,Contact Person's Office State,Contact Person's Office Zip Code,Federal Agency,Total Number of Entries Contained in Submission,Federal Taxpayer Identification Number,Legal Entity Identifier (LEI)
```
Comment (Kibrael) [inline docs/v2/spec/2018_Public_TS_Header.csv]: Please check for leading and trailing spaces.

File: docs/v2/spec/2018_Public_TS_Schema.csv (hunk 3 above)
Comment (MichaelFurey17) [reply docs/v2/spec/2018_Public_TS_Schema.csv]: Sure - I can change this.

File: docs/v2/spec/2018_Public_TS_Schema.csv (hunk 2 above)
Comment (MichaelFurey17) [reply docs/v2/spec/2018_Public_TS_Schema.csv]: Sure.

**Pull request merged**